Website/src/platform/App_Config/Include/Sitecore.Demo.CustomFields.config

```diff
@@ -26,9 +26,26 @@
                 <fieldType id="7">CmpMultiList</fieldType>
               </FieldTypes>
             </processor>
+
+            <!--<processor patch:after = "processor[@type='Sitecore.LayoutService.Serialization.Pipelines.GetFieldSerializer.GetMultilistFieldSerializer, Sitecore.LayoutService']" type="Sitecore.Demo.Edge.Website.CustomFields.CmpMultiListFIeldSerializer, Sitecore.Demo.Edge.Website" resolve="true">
+              <FieldTypes hint="list">
+                <fieldType id="1">CmpMultiList</fieldType>
+              </FieldTypes>
+            </processor>-->
+
           </getFieldSerializer>
         </pipelines>
       </group>
     </pipelines>
+
+    <services>
+
+      <register serviceType="Sitecore.Services.GraphQL.EdgeSchema.TemplateGeneration.FieldMapping.IFieldTypeFactoryStore, Sitecore.Services.GraphQL.EdgeSchema" implementationType="Sitecore.Services.GraphQL.EdgeSchema.TemplateGeneration.FieldMapping.DefaultFieldTypeFactoryStore, Sitecore.Services.GraphQL.EdgeSchema" patch:source="sitecore.services.graphql.edgecontent.config">
+        <patch:delete />
+      </register>
+
+      <register serviceType="Sitecore.Services.GraphQL.EdgeSchema.TemplateGeneration.FieldMapping.IFieldTypeFactoryStore, Sitecore.Services.GraphQL.EdgeSchema" implementationType="Sitecore.Demo.Edge.Website.CustomFields.DemoFieldTypeFactoryStore, Sitecore.Demo.Edge.Website" />
```

Comment:
Instead of a patch delete and adding a new one, we can patch the `implementationType` attribute:
```
<register serviceType="Sitecore.Services.GraphQL.EdgeSchema.TemplateGeneration.FieldMapping.IFieldTypeFactoryStore, Sitecore.Services.GraphQL.EdgeSchema">
  <patch:attribute name="implementationType">Sitecore.Demo.Edge.Website.CustomFields.DemoFieldTypeFactoryStore, Sitecore.Demo.Edge.Website</patch:attribute>
</register>
```